### PR TITLE
Update cubano manifest

### PR DESCRIPTION
**Pull request by jimmykemp**
Add entries to the MANIFEST.MF file. Helps to check which version is in
use, which is handy when  breaking API changes are introduced.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -14,6 +14,22 @@ apply plugin: 'nebula-aggregate-javadocs'
 apply plugin: 'org.ajoberstar.git-publish'
 apply plugin: 'net.researchgate.release'
 
+jar {
```
Comment (nigelcharman) [inline build.gradle]: This needs to be moved down to be inside the `subprojects` block below to be applied to all the subprojects.

The `versioning.info.commit` setting requires the `net.nemerosa.versioning` plugin:
* Add `classpath 'net.nemerosa:versioning:2.7.1'` to the `buildscript` > `dependencies` block at the top of the file.
* Add `apply plugin: 'net.nemerosa.versioning'` just below that block

File: build.gradle [hunk 2]
```diff
@@ -14,6 +14,22 @@ apply plugin: 'nebula-aggregate-javadocs'
 apply plugin: 'org.ajoberstar.git-publish'
 apply plugin: 'net.researchgate.release'
 
+task (type: Jar, overwrite: true) {
```
Comment (nigelcharman) [inline build.gradle]: This line's not working. I think it's creating a new unnamed task, rather than overriding the default task.

Comment (nigelcharman) [reply build.gradle]: See also comment above, it might show up as being on an outdated commit

Comment (nigelcharman) [reply build.gradle]: You can check if it is working using `gradlew pTML` to publish to maven local, then unzipping one of the jar files from `~/.m2/repository/org/concordion/cubano-core/0.2.2/cubano-core-0.2.2.jar` and checking the `META-INF/MANIFEST.MF` file.

**Comment by jimmykemp** [comment]
Righto, applied review changes.  Checked manifest following gradlew pTML, contains manifest variables.

**Pull request merged**